.github/workflows/build.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           name: plugin-artifact
-          path: ./build/distributions/${{ needs.build.outputs.artifact }}
```

Comment:
This happened to work as well as the `upload-artifact` action uploads the entire directory `./build/distributions/` which happens to contain just the plugin jar file.